Files/UserControls/ModernNavigationToolbar.xaml.cs

```diff
@@ -275,16 +275,16 @@ public async void CheckPathInput(ItemViewModel instance, string currentInput, st
                                 }
                             }
 
-                            if (!await Launcher.LaunchUriAsync(new Uri(currentInput)))
+                            try
                             {
-                                var dialog = new ContentDialog()
+                                if (!await Launcher.LaunchUriAsync(new Uri(currentInput)))
                                 {
-                                    Title = "Invalid item",
-                                    Content = "The item referenced is either invalid or inaccessible.\nMessage:\n\n" + ex.Message,
-                                    CloseButtonText = "OK"
-                                };
-
-                                await dialog.ShowAsync();
+                                    throw new Exception();
+                                }
+                            }
+                            catch
+                            {
```

Comment:
Use Ctrl + K + Ctrl + D shortcut to align code without such spaces.

Reply:
> Use Ctrl + K + Ctrl + D shortcut to align code without such spaces.

What is the alignment issue here??

![devenv_Fno8449MOs](https://user-images.githubusercontent.com/51203900/88482371-755bc980-cf7e-11ea-8d46-24f887f541a9.png)